### ISSUE TITLE
Sodium new version

### ISSUE THREAD

**Issue by farmani** [phpseclib/phpseclib#950]
SSH2 file has some bug here is a quick fix
line 1385
```php
$eBytes = \Sodium\crypto_box_publickey_from_secretkey($x);
```
line 1530 and 1531
```php
$key = new BigInteger(\Sodium\crypto_scalarmult($x, $fBytes), 256);
\Sodium\memzero($x);
```

obviously you should check the extension version and then applied the changes but I just drop a quick fix and bug report

**Comment by bantu** [comment]
Are you talking about :: vs \ ? That should be fixed already.

**Comment by farmani** [comment]
yes and method name also changed I check both master and 2 branch none of them changed to what I mentioned abow 
https://github.com/phpseclib/phpseclib/blob/master/phpseclib/Net/SSH2.php#L1397
https://github.com/phpseclib/phpseclib/blob/master/phpseclib/Net/SSH2.php#L1539
https://github.com/phpseclib/phpseclib/blob/master/phpseclib/Net/SSH2.php#L1540

**Comment by terrafrost** [comment]
Good catch. As of https://github.com/paragonie-scott/phpseclib/commit/fee6431badeeab42d2a59f94f365e345ab826705 phpseclib is already requiring libsodium 1.0.0+ so there's no need to check the version and do :: or \ depending on the version that's installed.

Anyway, this should now be fixed in the 2.0 and master branches.

Thanks!
